src/twisted/protocols/policies.py

```diff
@@ -712,6 +712,22 @@ def resetTimeout(self):
         if self.__timeoutCall is not None and self.timeOut is not None:
             self.__timeoutCall.reset(self.timeOut)
 
+
+    def cancelTimeout(self):
```

Comment:
On the one hand this is a new public API that isn't necessary; [passing `None` to `setTimeout` will cancel the timeout](https://twistedmatrix.com/documents/19.2.1/api/twisted.protocols.policies.TimeoutMixin.html#setTimeout).
On the other hand, `setTimeout`'s API is bad enough that a core committer with deep Twisted experience didn't see it, and I didn't either until I went hunting.  Maybe if we'd had this method in the first place we wouldn't have had this bug!
So I think I'm +0.75 for adding this method.
Should we deprecate the cancellation feature of `setTimeout`?